modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -126,7 +126,15 @@ const EditorManager: EditorManager = {
 
   baseURI: null as any,
   baseURL: null as any,
+
+  /**
+   * Default options for TinyMCE.
```

Comment:
There's not really a lot of explanation here. Is this really sufficient? What do `defaultOptions` do, for example? What is their relationship with options being passed through to `init`? Do we need to document that?